emission/analysis/modelling/trip_model/gradient_boosted_decision_tree.py

```diff
@@ -71,11 +63,12 @@ def fit(self, trips: List[ecwc.Confirmedtrip]):
         X_train, y_train = self.extract_features(trips)
         self.gbdt.fit(X_train, y_train)
         logging.info(f"gradient boosted decision tree model fit to {len(X_train)} rows of trip data")
+        logging.info(f"training features were {X_train.columns}")
 
-    def predict(self, trip: ecwc.Confirmedtrip) -> Tuple[List[Dict], int]:
+    def predict(self, trip: ecwc.Confirmedtrip) -> List:
```

Comment:
A list of trips can be passed (so that predict can be called on a batch of unlabeled trips all at once). So maybe this should be trip: List[ecwc.Confirmedtrip] -> List[string] ?

Reply:
Sure, processing a batch of trips at a time is definitely going to be more efficient than processing them one at a time.